.github/verify_header.sh

```diff
@@ -5,20 +5,30 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+COPYRIGHT_REGEX="Copyright\([Cc]\) ([0-9]{4}-([0-9]{4})|([0-9]{4}))[, ].*Intel Corporation$"
+LICENSE_REGEX="SPDX-License-Identifier: BSD-3-Clause$"
 YEAR=$(date +"%Y")
-REGEX="Copyright\(c\) [0-9]{4}-([0-9]{4}) |Copyright\(c\) ([0-9]{4}) "
+
+unset copyright_header license_header
 
 while read -r line; do
-    if [[ "$line" =~ $REGEX ]]; then
-        echo ${BASH_REMATCH[0]}
-        if [[ $YEAR == ${BASH_REMATCH[1]} || $YEAR == ${BASH_REMATCH[2]} ]]; then
-            echo $1 have appropriate license header
-            exit 0
+    [[ $copyright_header && $license_header ]] && break
+
+    if [[ "$line" =~ $COPYRIGHT_REGEX ]]; then
+        if [[ ${BASH_REMATCH[2]} == $YEAR || ${BASH_REMATCH[3]} == $YEAR ]]; then
+            copyright_header=1
+            continue
         fi
-        echo $1 have wrong license header year
-	exit 1
+        copyright_header=0
+        echo >&2 "error: wrong year in copyright info: '${BASH_REMATCH[0]}'"
```

Comment:
File may contain multiple "copyright" lines in random order, it's ok if any of them is up to date.